app/templates/direct-award/view-project.html

```diff
@@ -29,12 +34,21 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       {% with
-        heading = project.name
+        heading = project.name|default("Find cloud hosting, software and support")
       %}
         {% include "toolkit/page-heading.html" %}
       {% endwith %}
     </div>
   </div>
+  {% if not project %}
+    <p class="lede">
```

Comment:
I don't even think this *is* a lede, so let's just choose a different word that isn't quite so obscure?

Style-wise: can't it just be inside a `column-two-thirds`? Does it really need a style of its own?

Reply:
I haven't chosen the name - it comes from a style in the govuk toolkit which we don't have for some reason and the scss itself is copypasta'd from ... the supplier app?

Reply:
Ouch well it ought to go into our frontend toolkit then, but whatevs.

Reply:
Kinda, maybe, eventually, or maybe we get around to doing design system stuff first...